tests/framework/utils/k8s_helper.go

```diff
@@ -644,3 +646,43 @@ func (k8sh *K8sHelper) IsRookInstalled() bool {
 
 	return false
 }
+
+//GetRookLogs captures logs from specified rook pod and writes it to specified file
+func (k8sh *K8sHelper) GetRookLogs(podAppName string, namespace string, filePath string, testName string) {
+	lines := int64(2000)
+	logOpts := &v1.PodLogOptions{TailLines: &lines}
```

Comment:
I will add the full logs, since we install and uninstall rook for every test suite, the log should be pretty small. I was planning on getting the logs based on timestamp( from test start time),but the k8s api looks buggy, I cannot retrieve logs from pods in default pod if I use timestamp options.